SoundSwitch/Framework/NotificationManager/NotificationManager.cs

```diff
@@ -84,14 +82,6 @@ private void SetNotification(NotificationTypeEnum notificationTypeEnum)
             }
             catch (CachedSoundFileNotExistsException)
             {
-                if (!_notification.NeedCustomSound())
-                {
-                    return;
-                }
-
-                MessageBox.Show(string.Format(SettingsStrings.audioFileNotFound, SettingsStrings.notificationOptionSound),
```

Comment:
Might want to keep this.

This is to tell the user we couldn't set the file to the path provided.